File: help/pt_pt/exercicios_da_ficha_9/Ficha09_006/Ficha09_006.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Limite inferior : ");
    int inf;
    scanf("%d", &inf);
    if(inf % 2 == 0 ){
        inf++;
    }
    printf("Limite superior : ");
    int sup;
    scanf("%d", &sup);
    while(inf <= sup ){
        printf("%d ",inf );
        inf +=2;
    }
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:25

File: help/pt_pt/exercicios_da_ficha_9/Ficha09_006/Ficha09_006.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha09_006 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Limite inferior : " );
        int inf = Integer.valueOf(keyb.nextLine());
        if(inf % 2 == 0 ){
            inf++;
        }
        System.out.print("Limite superior : " );
        int sup = Integer.valueOf(keyb.nextLine());
        while(inf <= sup ){
            System.out.print(inf + " " );
            inf +=2;
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:25
